src/apps/ontology/sc/opcode.py

```diff
@@ -0,0 +1,151 @@
+# OpCode constants
```

Comment:
we don't need a full list of opcodes. Please reduce this only to opcodes that are actually used in the code.